scanner/services/converters/converters.go

```diff
@@ -22,6 +23,7 @@ var (
 		claircore.High:       v4.VulnerabilityReport_Vulnerability_SEVERITY_IMPORTANT,
 		claircore.Critical:   v4.VulnerabilityReport_Vulnerability_SEVERITY_CRITICAL,
 	}
+	vulnNamePattern = regexp.MustCompile(`((CVE|ALAS|DSA)-\d{4}-\d+)|((RHSA|RHBA|RHEA)-\d{4}:\d+)`)
```

Comment:
Just checked, and ALAS2 is another possibility as well as ALAS2023 (and I imagine an ALAS20XY for each year afterwards)

Reply:
Oracle has ELSA (maybe others?)

Reply:
I think it may be best to generalize this, even though the regex evaluation may be slower. 

maybe something like `[A-Z]+-\d{4}[-:]\d+`? very generic but at least covers all the known formats

Reply:
> Are you sure this covers all possibilities?

No, I am not. This can be expanded and added towards being complete. This is a kick-off.

> I think it may be best to generalize this, even though the regex evaluation may be slower.

In my opinion, using a general regex increases the risk of false matches, especially since we are using the same regex for titles and links. This can lead to inadvertent matches. Instead, I suggest creating a list of potential matches based on the security sources we plan to support. We can then collaborate with ClairCore to develop a specialized field in the vulnerability report that supports these vulnerability names, or similar alternative that involves ClairCore parsing the name in the appropriate place. This approach will ensure more accurate results and reduce the risk of false matches.

Reply:
At least the ones I know:

* CVE
* RHSA/RHBA/RHEA
* GHSA
* ALAS/ALAS2/ALAS2023/etc

I do not believe we show DSA nor USN, but I could be wrong. We can sync offline about this to figure out the known formats

Reply:
We can go incrementally. I will add the ones you are suggesting.

Reply:
I think it makes sense to prioritize RHSA/RHBA/RHEA over CVE. CVE over GHSA. Why ALAS over CVE?

Also, please provide the reference where you found ALAS2 and ALAS2023 (without a hyphen).

Reply:
Searching the web, the security advisory ID format for Amazon Linux AMI seems to follow the pattern ALAS-YYYY-NNNN.